README.md

```diff
@@ -1,50 +1,41 @@
 # Forte EpiServer Azure Search
 
-This code ads search functionality to your EpiServer project based on Azure Cognitive Search service
+This code adds search functionality to your EpiServer project based on Azure Cognitive Search service
 
 ## Configuration
 
-Once you create Azure Cognitive Search service you'll get service name and admin keys. 
-You need to define those two in your appSettings:
-
-**web.config**
-
-```xml
- <appSettings>
-    <add key="AzureSearchService:Name" value="yourservicename" />
-    <add key="AzureSearchService:ApiKey" value="YOURADMINKEY" />
-  </appSettings>
-``` 
+Once you create Azure Cognitive Search service you'll get service name and admin keys.
 
 NOTE: service name is without _search.windows.net_ suffix.
 
 ## Initialization
 
-For basic search initialization it's enough to create class like this:
+For basic search initialization it's enough to add code like this:
 
+In your **Startup.cs** file
+
+_Step 1._
 ```c#
-using System.Configuration;
-using EPiServer.Framework;
-using Forte.EpiServer.AzureSearch.Configuration;
+public void ConfigureServices(IServiceCollection services)
+{
+    // (...)
+    services.AddEpiServerAzureSearch<ContentDocument, DefaultDocumentBuilder>("yourservicename", "YOURADMINKEY"); 
+    // (...)
+}
+```
+You need to pass `AzureSearchServiceConfiguration` object with your service name and the admin key. 
```

Comment:
Hmm in the given example you provide name and key without wrapping it with this Configuration object - mistake?

Reply:
We decided to remove this object and use strings directly as a required parameters

Reply:
so the readme is wrong :D